### PR TITLE
Add FrameworkColour

### PR DESCRIPTION
**Pull request by jorolf**
I'm unsure about the colours names, the design only had a palette in the top right corner.

### CODE REVIEW

**Comment by peppy** [comment]
Might sit better at just the `Graphics` namespace.

File: osu.Framework/Graphics/FrameworkColour.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osuTK.Graphics;
+
+namespace osu.Framework.Graphics
+{
+    public static class FrameworkColour
+    {
+        public static Color4 Blue => new Color4(40, 65, 82, 255);
```
Comment (peppy) [inline osu.Framework/Graphics/FrameworkColour.cs]:
```suggestion
        public static Color4 Blue { get; } = new Color4(40, 65, 82, 255);
```

maybe feels better? not sure it changes much. @smoogipoo opinions?

Comment (smoogipoo) [reply osu.Framework/Graphics/FrameworkColour.cs]: What are you trying to achieve?

Comment (peppy) [reply osu.Framework/Graphics/FrameworkColour.cs]: Just asking your preferred code style.

**Pull request merged**